x-pack/platform/plugins/shared/fleet/public/components/header.tsx

```diff
@@ -35,21 +35,14 @@ const Wrapper = styled.div<{ maxWidth?: number | string }>`
   padding-right: ${(props) => props.theme.eui.euiSizeM};
 `;
 
-const Tabs = styled(EuiTabs)`
-  top: 1px;
-  &:before {
-    height: 0px;
-  }
-`;
-
 export interface HeaderProps {
   maxWidth?: number | string;
   leftColumn?: JSX.Element;
   rightColumn?: JSX.Element;
   rightColumnGrow?: EuiFlexItemProps['grow'];
   topContent?: JSX.Element;
   tabs?: Array<Omit<EuiTabProps, 'name'> & { name?: JSX.Element | string }>;
-  tabsClassName?: string;
+  tabsCss?: string;
```

Comment:
nit: will it be better to type this as an emotion interpolation as type here? and use emotion combination after?

Reply:
i tried a few ways of doing this including emotion interpolation here, but couldn't get it to work with the additional styles that we already add to `EuiTabs` 😔